packages/flutter_tools/ide_templates/intellij/.idea/runConfigurations/manual_tests___focus.xml.copy.tmpl

```diff
@@ -0,0 +1,6 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="manual_tests - focus" type="FlutterRunConfigurationType" factoryName="Flutter" singleton="false">
+    <option name="filePath" value="$PROJECT_DIR$/dev/manual_tests/lib/focus.dart" />
+    <method v="2" />
+  </configuration>
+</component>
```

Comment:
?

Reply:
This adds a run configuration to IntelliJ for the focus manual test I added.

Reply:
huh. ok :-)